net.meijn.onvifviewer.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "net.meijn.onvifviewer",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "onvifviewer",
+    "finish-args": [
+        "--socket=x11",
+        "--share=network"
+    ],
+    "modules": [
```

Comment:
You install lots of modules but are missing a `cleanup` for all the headers and whatnot installed.